src/util/NamedColor.cpp

```diff
@@ -26,9 +26,11 @@ auto operator>>(std::istream& str, NamedColor& namedColor) -> std::istream& {
     NamedColor tmp;
     if (std::getline(str, line)) {
         std::istringstream iss{line};
+        iss.imbue(std::locale::classic());
```

Comment:
Maybe add a comment on why this is needed as it might be a bit surprising here on first sight.